coreneuron/io/nrn_setup.cpp

```diff
@@ -528,6 +512,28 @@ void nrn_setup(const char* filesdat,
     /* nrn_multithread_job supports serial, pthread, and openmp. */
     coreneuron::phase_wrapper<coreneuron::phase::two>(userParams, corenrn_embedded);
 
+    // gap junctions
+    // Gaps are done after phase2, in order to use layout and permutation
+    // information via calls to stdindex2ptr.
+    if (nrn_have_gaps) {
+        nrn_partrans::transfer_thread_data_ = new nrn_partrans::TransferThreadData[nrn_nthread];
```

Comment:
@nrnhines : I couldn't see where this `transfer_thread_data_ ` is deallocated. could you check?